config_linux.go

```diff
@@ -251,6 +251,20 @@ type Device struct {
 	GID uint32 `json:"gid"`
 }
 
+// DeviceCgroup represents a device rule for the whitelist controller
+type DeviceCgroup struct {
+	// Allow or deny
+	Allow bool `json:"allow"`
+	// Device type, block, char, etc.
+	Type *rune `json:"type,omitempty"`
+	// Major is the device's major number.
+	Major *int64 `json:"major,omitempty"`
+	// Minor is the device's minor number.
+	Minor *int64 `json:"minor,omitempty"`
+	// Cgroup access permissions format, rwm.
+	Access string `json:"access"`
```

Comment:
On Wed, Jan 27, 2016 at 01:16:29PM -0800, Vish Kannan wrote:
> > +	// Cgroup access permissions format, rwm.
> > +	Access string `json:"access"`
> 
> AFAIK, this field must be optional

I'm not sure about *must* (when would you not be able to supply a
value?), but I'm fine making it optional, defaulting to not writing an
access entry to the cgroup device file.  The filesystem API allows
[1]:

  $ echo a > /sys/fs/cgroup/1/devices.deny

which acts like:

  $ echo 'a *:* rwm' > /sys/fs/cgroup/1/devices.deny

so they have an implicit access there (although I don't see
documentation for that beyond the example quoted above).  

[1]: https://kernel.org/doc/Documentation/cgroup-v1/devices.txt